mesonbuild/scripts/vcstagger.py

```diff
@@ -14,24 +14,30 @@
 
 import sys, os, subprocess, re
 
+
 def config_vcs_tag(infile, outfile, fallback, source_dir, replace_string, regex_selector, cmd):
     try:
         output = subprocess.check_output(cmd, cwd=source_dir)
         new_string = re.search(regex_selector, output.decode()).group(1).strip()
     except Exception:
         new_string = fallback
 
-    with open(infile) as f:
+    kwargs = {}
+    if sys.version_info.major >= 3:
```

Comment:
Meson only supports Python 3, this check is unnecessary.

Reply:
ok so the fix is easier, we have just to put `encoding='utf8'` each time we call the `open` function.
So I will do a rebase in order to provide a clean history.